Index: ../.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ExternalStorageConfigurationManager\" enabled=\"true\" />\r\n  <component name=\"MavenProjectsManager\">\r\n    <option name=\"originalFiles\">\r\n      <list>\r\n        <option value=\"$PROJECT_DIR$/LockedIn/pom.xml\" />\r\n        <option value=\"$PROJECT_DIR$/Resources/tusar-research/r&amp;d - week 5 checkpoint/springBoot/pom.xml\" />\r\n      </list>\r\n    </option>\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_24\" project-jdk-name=\"24\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../.idea/misc.xml b/../.idea/misc.xml
--- a/../.idea/misc.xml	(revision d0972d212aaba99c4eb29973df1ebea6daf5b1c1)
+++ b/../.idea/misc.xml	(date 1777338075538)
@@ -1,15 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="MavenProjectsManager">
-    <option name="originalFiles">
-      <list>
-        <option value="$PROJECT_DIR$/LockedIn/pom.xml" />
-        <option value="$PROJECT_DIR$/Resources/tusar-research/r&amp;d - week 5 checkpoint/springBoot/pom.xml" />
-      </list>
-    </option>
-  </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_24" project-jdk-name="24" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_25" project-jdk-name="corretto-21" project-jdk-type="JavaSDK" />
 </project>
\ No newline at end of file
Index: ../.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/CAB302_Project.iml\" filepath=\"$PROJECT_DIR$/.idea/CAB302_Project.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/LockedIn/LockedIn.iml\" filepath=\"$PROJECT_DIR$/LockedIn/LockedIn.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../.idea/modules.xml b/../.idea/modules.xml
--- a/../.idea/modules.xml	(revision d0972d212aaba99c4eb29973df1ebea6daf5b1c1)
+++ b/../.idea/modules.xml	(date 1777338075477)
@@ -2,7 +2,6 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/CAB302_Project.iml" filepath="$PROJECT_DIR$/.idea/CAB302_Project.iml" />
       <module fileurl="file://$PROJECT_DIR$/LockedIn/LockedIn.iml" filepath="$PROJECT_DIR$/LockedIn/LockedIn.iml" />
     </modules>
   </component>
